x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -1028,6 +1036,104 @@ export class AlertsClient {
     }
   }
 
+  /**
+   * Performs a `find` query to extract aggregations on alert groups
+   */
+  public getGroupAggregations({
```

Comment:
How about adding something like `x-pack/plugins/rule_registry/server/alert_data_client/tests/getGroupAggregations.test.ts` that tests this?

Nothing too elaborate, to ensure that `find` is called with the correct params and no changes break this function.

I know we do some testing in `x-pack/plugins/rule_registry/server/routes/get_alerts_group_aggregations.test.ts` but that is like "indirectly" 😛

Reply:
+1

Reply:
+1